cloudmarker/test/test_main.py

```diff
@@ -8,7 +8,8 @@
 class MainTest(unittest.TestCase):
     """Tests for package execution."""
 
-    @mock.patch('sys.argv', ['cloudmarker', '-c', 'config.base.yaml', '-n'])
+    @mock.patch('sys.argv', ['cloudmarker', '-n', '-c',
+                             'cloudmarker.base.yaml'])
```

Comment:
This works without the 4th argument `cloudmarker.base.yaml`.

Reply:
Good catch. [Fixed here](https://github.com/cloudmarker/cloudmarker/pull/90/files#diff-3cfc46b1f4ad6f67f74d79216e9c6eb5R11).